lib/plugins/bed.js

```diff
@@ -55,20 +56,43 @@ function inject (bot) {
       cb(new Error('wrong block : not a bed block'))
     } else {
       const metadata = parseBedMetadata(bedBlock)
-      let headBlock = bedBlock
+      let headPoint = bedBlock.position
+
+      if (metadata.occupied === 1) return cb(new Error('the bed is occupied'))
```

Comment:
In my opinion single line if statements are kind of unreadable but it's up to you if standard isn't throwing an error there